extensions/dou-tu/README.md

```diff
@@ -0,0 +1,9 @@
+# Dou Tu Extension for Raycast
+
+上班时间斗个图 😄
```

Comment:
At the moment, Raycast doesn't support localization and only supports US English. Therefore, please avoid introducing your custom way to localize your extension.